EvalEngine/EvalEngineImpl.cpp

```diff
@@ -183,7 +183,9 @@ void EvalEngineImpl::evaluate(void)
 
     //Do not evaluate when there are pending events in the queue.
     //Evaluate only after all events received - AKA event compression.
+    #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
     if (this->thread()->eventDispatcher()->hasPendingEvents()) return;
+    #endif
```

Comment:
This missing call needs more consideration than ifdef away. A shared object with a ref count could work for knowing if more events are in-flight.